app.py

```diff
@@ -15,40 +15,60 @@ def index():
     # If form is submitted
     if request.method == 'POST':
         NotamRequest = Models.NotamRequest(request.form)
+        airports = [NotamRequest.startAirport, NotamRequest.destAirport]
+        for destination in NotamRequest.destinations:
+            airports.append(destination)
+        apiOutputs = []
 
-        # get lat/long of airports
-        NotamRequest.startLat, NotamRequest.startLong = alc.get_lat_and_lon(NotamRequest.startAirport)
-        NotamRequest.destLat, NotamRequest.destLong = alc.get_lat_and_lon(NotamRequest.destAirport)
-        
-        NotamRequest.radius = int(NotamRequest.radius)
-        NotamRequest.pathWidth = int(NotamRequest.pathWidth)
-
-        # get the list of coordinates that need to be called to cover area
-        coordList = MinimalCirclesPath.getPath(NotamRequest.startLat, 
-                                               NotamRequest.startLong,
-                                               NotamRequest.destLat,
-                                               NotamRequest.destLong, 
-                                               NotamRequest.radius, # circle radius
-                                               NotamRequest.pathWidth) # path width
+        #i is used to track where we are in the array of airports
+        i = 0
         # start timer
         startTime = time.time() 
+        
+        #len(airports) -1 ensures the loop treats i as the starting location for each iteration and prevents out of index errors
+        while i < len(airports) - 1:
+            
+            # get lat/long of airports
+            NotamRequest.startLat, NotamRequest.startLong = alc.get_lat_and_lon(airports[i])
+            NotamRequest.destLat, NotamRequest.destLong = alc.get_lat_and_lon(airports[i+1])

```

Comment:
if we keep updating the start and dest then when we send that information to the view our start would be the second to last location rather than the actual start. You could use a temporary variable while you loop to fix this